common/test/acceptance/tests/video/test_video_module.py

```diff
@@ -1223,7 +1224,7 @@ def test_if_captions_are_draggable(self):
             )
         else:
             self.assertEqual(
-                captions_end.get('y') + 15,
+                captions_end.get('y') + 16,
```

Comment:
Hey @andy-armstrong what was the context behind this change? I am getting bokchoy failure on my PR #15247 which would get pass if we do `captions_end.get('y') + 15` – Failure message says: 689.0 != 688.0 Closed captions did not get dragged.